src/main/java/com/karankumar/bookproject/ui/book/BookForm.java

```diff
@@ -47,6 +48,8 @@
 import java.util.Objects;
 import java.util.logging.Level;
 
+import static com.karankumar.bookproject.ui.FormErrors.*;
```

Comment:
We're not using wildcard imports (see https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)